README.md

```diff
@@ -51,9 +51,12 @@ let package = Package(
 | `swiftformat.options`                     | `[String]` | `[]`               | Additional [options for SwiftFormat](https://github.com/nicklockwood/SwiftFormat#options). |
 | `swiftformat.configSearchPaths`           | `[String]` | `[".swiftformat"]` | Possible paths for SwiftFormat config.                                                     |
 
-## Contributors
```

Comment:
In case you wish we can also keep the contributors section. Just felt like we don't need it, especially as it was quite outdated.